src/CoreWCF.Primitives/src/CoreWCF/Description/DispatcherBuilder.cs

```diff
@@ -647,6 +647,11 @@ private static void BindOperations(ContractDescription contract, ClientRuntime p
                             IOperationBehavior behavior = operation.Behaviors[j];
                             behavior.ApplyDispatchBehavior(operation, dispatchOperation);
                         }
+                        for (int k = 0; k < operation.AuthorizeOperation.Count; k++)
+                        {
```

Comment:
Blank line between for blocks. Also I think foreach would result in more compact code